### PR TITLE
feat: seo pages

### PR DESCRIPTION
**Pull request by oktapodia**

### CODE REVIEW

File: src/app/bridge/layout.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
+import React from 'react';
```
Comment (oktapodia) [inline src/app/bridge/layout.tsx:1]: Might change depending of Eugene progress

File: src/app/ui/bridge/BridgePage.tsx [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+        </Typography>
+        <Widget
+          isWelcomeScreenClosed={true}
+          starterVariant="default"
+          fromChain={sourceChain?.id}
+          toChain={destinationChain?.id}
+          fromToken={sourceToken?.address}
+          toToken={destinationToken?.address}
+        />
+        <BridgePageContainer sx={(theme) => ({ marginTop: theme.spacing(4) })}>
```
Comment (oktapodia) [inline src/app/ui/bridge/BridgePage.tsx:45]: Content needs to be reviewed

File: next.config.mjs [hunk 3]
```diff
@@ -53,6 +47,48 @@ const nextConfig = {
         port: '',
         pathname: '/**',
       },
+      {
+        protocol: 'https',
+        hostname: 'raw.githubusercontent.com', // TODO: this one can be dangerous
```
Comment (tcheee) [inline next.config.mjs:52]: to remove when we move on prod

File: src/app/ui/bridge/BridgePage.tsx [hunk 4]
```diff
@@ -0,0 +1,304 @@
...
+          <Typography
```
Comment (tcheee) [inline src/app/ui/bridge/BridgePage.tsx:51]: We should extract inside its own component with all the text.

Comment (tcheee) [reply src/app/ui/bridge/BridgePage.tsx:51]: same feedback

File: src/app/ui/bridge/BridgePage.tsx [hunk 5]
```diff
@@ -0,0 +1,304 @@
...
+            variant="h1"
```
Comment (tcheee) [inline src/app/ui/bridge/BridgePage.tsx:52]: should be <h2> and 36px

File: src/app/ui/bridge/HalfSizeBlock.tsx [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+          <Box display="flex" marginRight={2}>
+            <Avatar
+              variant="rounded"
+              src={info.logoURI}
+              alt={info.name}
+              sx={{ width: 32, height: 32, borderRadius: 50 }}
+            />
+          </Box>
+        )}
+        {info.name} - {type} Information
```
Comment (tcheee) [inline src/app/ui/bridge/HalfSizeBlock.tsx:64]: we should keep only the symbol if we cant have only the token Name

File: src/app/ui/bridge/BridgePage.tsx [hunk 7]
```diff
@@ -0,0 +1,304 @@
...
+              info={{
+                logoURI: token?.logoURI,
+                name: `${token.name} (${token.symbol})`,
+              }}
+              data={getTokenInfoData(chains, token)}
+            />
+          ))}
+        </Stack>
+        <BridgePageContainer>
+          <Typography variant="h3" marginY={2}>
```
Comment (tcheee) [inline src/app/ui/bridge/BridgePage.tsx:186]: this should be <h3 but 32px

File: src/app/ui/bridge/BridgePage.tsx [hunk 8]
```diff
@@ -0,0 +1,304 @@
...
+          </Typography>
+          <Typography>
+            To bridge from {sourceToken?.name} on {sourceChain?.name} to{' '}
+            {destinationToken?.name} on {destinationChain?.name}, you will need
+            to follow these steps. This guide will walk you through the process
+            of transferring your assets from {sourceToken?.name} on{' '}
+            {sourceChain?.name} to {destinationToken?.name} on{' '}
+            {destinationChain?.name}.
+          </Typography>
+          <Typography variant="h2" marginY={2}>
```
Comment (tcheee) [inline src/app/ui/bridge/BridgePage.tsx:68]: this should be <h3 and 24px

File: src/app/ui/bridge/BridgePage.tsx [hunk 9]
```diff
@@ -0,0 +1,304 @@
...
+            use it on Avalanche. Since these two networks operate independently
+            with their own rules and consensus mechanisms, direct communication
+            isn't possible. To use her ETH on Avalanche, Alice can utilize a
+            blockchain bridge to convert her ETH into wETH (wrapped ETH) that
+            can function on Avalanche. This process allows her to access the
+            features of both blockchains without needing to acquire additional
+            assets.
+          </Typography>
+
+          <Typography variant="h4" marginY={2}>
```
Comment (tcheee) [inline src/app/ui/bridge/BridgePage.tsx:212]: image
should be the same size at the various 'Isolation' / 'Interoperability' below

File: src/app/bridge/layout.tsx (hunk 1 above)
Comment (tcheee) [reply src/app/bridge/layout.tsx:1]: what abou this @oktapodia ?

**Review by tcheee — APPROVED** [review]

**Review by dennyscode — APPROVED** [review]
So, so so sweet!

File: next.config.mjs (hunk 3 above)
Comment (oktapodia) [reply next.config.mjs:58]: We unfortunately cannot because of some tokens image that we get from this (mainly trustwallet)

**Pull request merged**